typescript/packages/wallets/crossmint/src/custodial.ts

```diff
@@ -84,9 +85,10 @@ export function custodialFactory(apiKey: string) {
             },
             async sendTransaction({ instructions }: SolanaTransaction) {
                 const latestBlockhash = await connection.getLatestBlockhash("confirmed");
```

Comment:
Doesn't crossmint override the latest blockchash value in the server?

Reply:
(Confirmed offline it does)

Given this, do we still need to pass around a connection object? Seems easier to cut it out

Reply:
We do, generating cNFT transactions requires a connection as you have to fetch merkle tree info